pkg/cluster/internal/create/actions/kubeedge/kubeedge.go

```diff
@@ -326,31 +326,39 @@ func (a *Action) runStartEdgecore(ctx *actions.ActionContext, node nodes.Node) e
 		return fmt.Errorf("failed to generate cloudcore config: %v", err)
 	}
 
-	cmd = node.Command("bash", "-c", fmt.Sprintf(`sed -i -e "s|token: .*|token: %s|g" /etc/kubeedge/config/edgecore.yaml`, KubeEdgeToken))
+	cmd = node.Command("bash", "-c", fmt.Sprintf(`sed -i -e "/metaServer:/{n;n;s/false/true/;}" /etc/kubeedge/config/edgecore.yaml`))
 	lines, err = exec.CombinedOutputLines(cmd)
 	ctx.Logger.V(3).Info(strings.Join(lines, "\n"))
 	if err != nil {
-		return fmt.Errorf("failed to modify token: %v", err)
+		return fmt.Errorf("failed to modify metaServer: %v", err)
+	}
+
+	cmd = node.Command("bash", "-c", fmt.Sprintf(`sed -i -e "s|runtimeType: .*|runtimeType: remote|g" /etc/kubeedge/config/edgecore.yaml`))
+	lines, err = exec.CombinedOutputLines(cmd)
+	ctx.Logger.V(3).Info(strings.Join(lines, "\n"))
+	if err != nil {
+		return fmt.Errorf("failed to modify runtimeType: %v", err)
 	}
 
-	// modify runtime to containerd
-	cmd = node.Command("bash", "-c", fmt.Sprintf(`sed -i -e "s|remoteImageEndpoint: .*|remoteImageEndpoint: %s|g" /etc/kubeedge/config/edgecore.yaml`, "unix:///var/run/containerd/containerd.sock"))
+	cmd = node.Command("bash", "-c", fmt.Sprintf(`sed -i -e "s|cgroupDriver: .*|cgroupDriver: systemd|g" /etc/kubeedge/config/edgecore.yaml`))
```

Comment:
Why modify to systemd here?

Reply:
I referred to an issue to resolve compatibility problems.
If not, it will error.